server/app/services/ai_enrichment.py

```
@@ -0,0 +1,161 @@
+            ai_notes = parsed.get("ai_notes") or ""
+            if isinstance(ai_notes, list):
+                ai_notes = "\n".join([str(item) for item in ai_notes])
+            ai_notes = str(ai_notes)
+
+            # Sanitize technical_specs
+            specs = parsed.get("technical_specs")
+            if isinstance(specs, str):
+                try:
+                    import json
+                    specs = json.loads(specs)
+                except:
```

Ruff (0.15.12)

[error] Do not use bare `except`

(E722)